File: 백준/Bronze/10250. ACM 호텔/ACM 호텔.cc
#include<stdio.h>

int main() {
    int H, W, N, T;
    scanf("%d", &T);
    for (int i=0; i < T; i++){
        scanf("%d %d %d", &H, &W, &N);
        int last = N % H ? N / H + 1 : N / H;
        int first = N % H ? N % H : H;
        printf("%d\n", first * 100 + last);
    }
    return 0;
}

File: 백준/Bronze/10250. ACM 호텔/ACM 호텔.java
import java.util.Scanner;

public class Main {
    public static void main(String args[]) {
    int H, W, N, T;
    Scanner s = new Scanner(System.in);
    T = s.nextInt();
    for (int i=0; i < T; i++){
        H = s.nextInt();
        W = s.nextInt();
        N = s.nextInt();
        int last = N % H != 0 ? N / H + 1 : N / H;
        int first = N % H != 0 ? N % H : H;
        System.out.printf("%d\n", first * 100 + last);
    }
    }
}